src/pybind/mgr/balancer/module.py

```diff
@@ -685,6 +689,7 @@ def do_crush_compat(self, plan):
                 else:
                     key = 'bytes'
 
+            # Convergence Testing Algorithm
```

Comment:
There was another instance where I used literal string for commenting out the code, but the branch has been merged.
Should I change the commenting style there too?